src/components/ChildBountyCreation/CreateChildBounty.tsx

```
@@ -0,0 +1,379 @@
+			apiReady,
+			errorMessageFallback: 'Transaction failed.',
+			network,
+			onFailed,
+			onSuccess,
+			setStatus: (message: string) => setLoadingStatus({ isLoading: true, message: message }),
+			tx: txDetails.tx
+		});
+	};
+
+	useEffect(() => {
+		if ((!parentBountyIndex || isNaN(parentBountyIndex)) && parentBountyIndex !== null) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)